logger/logger_config.go

```diff
@@ -25,7 +25,7 @@ type Level string
 const (
 	// OffLevel disables logging.
 	OffLevel Level = "off"
-	// CriticalLevel level. Used for errors that should definitely be noted.
+	// CriticalLevel is used for errors that halts the client.
 	CriticalLevel Level = "critical"
 	// ErrorLevel level. Logs. Used for errors that should definitely be noted.
```

Comment:
Can you use the same document format for other log level doc comments? `ErrorLevel is used ...`, etc.